.ci/magic-modules/generate-ansible.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# This script takes in 'magic-modules-branched', a git repo tracking the head of a PR against magic-modules.
+# It outputs "ansible-generated", a non-submodule git repo containing the generated ansible code.
+
+set -x
+set -e
+
+pushd magic-modules-branched
+LAST_COMMIT_AUTHOR="$(git log --pretty="%an <%ae>" -n1 HEAD)"
+bundle install
+bundle exec compiler -p products/compute -e ansible -o "build/ansible/"
+
+# This command can crash - if that happens, the script should not fail.
+set +e
+ANSIBLE_COMMIT_MSG="$(python .ci/magic-modules/extract_from_pr_description.py --tag ansible < .git/body)"
```

Comment:
Remember to add the `ansible` tag to the .github/PULL_REQUEST_TEMPLATE.md.

Reply:
A document for how to add a new provider is desperately needed. Right now is beyond witchcraft how to add a new product or provider. Any chance we can do that?

Reply:
I have an issue to write that up, but it's not near the top of my list right now.  Do you think it should be?